test/micro/org/openjdk/bench/jdk/incubator/vector/MaskQueryOperationsBenchmark.java

```diff
@@ -0,0 +1,159 @@
+@State(Scope.Thread)
+public class MaskQueryOperationsBenchmark {
+    @Param({"128","256","512"})
+    int bits;
+
+    @Param({"1","2","3"})
+    int inputs;
+
+    VectorSpecies bspecies;
+    VectorSpecies sspecies;
+    VectorSpecies ispecies;
+    VectorSpecies lspecies;
+    VectorMask    bmask;
+    VectorMask    smask;
+    VectorMask    imask;
+    VectorMask    lmask;
+    boolean []    mask_arr;
+
+
+    static final boolean [] mask_avg_case = {
+       false, false, false, true, false, false, false, false,
+       false, false, false, true, false, false, false, false,
+       false, false, false, true, false, false, false, false,
+       true, true, true, true, true, true, true, true,
+       true, true, true, true, true, true, true, true,
+       false, false, false, true, false, false, false, false,
+       false, false, false, true, false, false, false, false,
+       false, false, false, true, false, false, false, false
+    };
+
+    static final boolean [] mask_best_case  = {
+       true, true, true, true, true, true, true, true,
+       true, true, true, true, true, true, true, true,
+       true, true, true, true, true, true, true, true,
+       true, true, true, true, true, true, true, true,
+       true, true, true, true, true, true, true, true,
+       true, true, true, true, true, true, true, true,
+       true, true, true, true, true, true, true, true,
+       true, true, true, true, true, true, true, true
+    };
+
+    static final boolean [] mask_worst_case  = {
+       false, false, false, false, false, false, false, false,
+       false, false, false, false, false, false, false, false,
+       false, false, false, false, false, false, false, false,
+       false, false, false, false, false, false, false, false,
+       false, false, false, false, false, false, false, false,
+       false, false, false, false, false, false, false, false,
+       false, false, false, false, false, false, false, false,
+       false, false, false, false, false, false, false, false
+    };
+
+    @Setup(Level.Trial)
+    public void BmSetup() {
+        bspecies = VectorSpecies.of(int.class, VectorShape.forBitSize(bits));
+        sspecies = VectorSpecies.of(int.class, VectorShape.forBitSize(bits));
+        ispecies = VectorSpecies.of(int.class, VectorShape.forBitSize(bits));
+        lspecies = VectorSpecies.of(long.class, VectorShape.forBitSize(bits));
+
+        if( 1 == inputs) {
+          mask_arr = mask_best_case;
+        } else if ( 2 == inputs ) {
+          mask_arr = mask_worst_case;
+        } else {
+          mask_arr = mask_avg_case;
+        }
+
+        bmask   = VectorMask.fromArray(bspecies, mask_arr, 0);
+        smask   = VectorMask.fromArray(sspecies, mask_arr, 0);
+        imask   = VectorMask.fromArray(ispecies, mask_arr, 0);
+        lmask   = VectorMask.fromArray(lspecies, mask_arr, 0);
+    }
+
+    @Benchmark
+    public void testTrueCountByte(Blackhole bh) {
+        bh.consume(bmask.trueCount());
+    }
```

Comment:
No need to use a black hole. A returned value will be consumed by a black hole by the framework.
 
```suggestion
    @Benchmark
    public int testTrueCountByte() {
        return bmask.trueCount();
    }
```